101-azure-spring-cloud/azuredeploy.json

```diff
@@ -0,0 +1,183 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceName": {
+            "type": "string"
+        },
+        "app1": {
+            "type": "string",
+            "defaultValue": "gateway"
+        },
+        "app2": {
+            "type": "string",
+            "defaultValue": "account-service"
+        },
+        "app3": {
+            "type": "string",
+            "defaultValue": "auth-service"
+        },
+        "location": {
+          "type": "string",
+          "defaultValue": "[resourceGroup().location]"
+        },
+        "location-spring-cloud": {
```

Comment:
remove this and just use the location parameter... do not add the defaultValue, set it in the parameter file.